lib/dry/cli/command.rb

```diff
@@ -72,7 +100,9 @@ def self.extended(base)
       #     end
       #   end
       def self.desc(description)
-        @description = description
+        @_mutex.synchronize do
```

Comment:
this is not an array or hash, we just assing a value, should we wrap it with synchronize?

Reply:
Looks like it's necessary:

```ruby
class Counter
  def self.set(n)
    @n = n
  end

  def self.get
    @n
  end
end

test = proc do
  threads = 100.times.map { |n| Thread.new { Counter.set(n.next) } }
  threads.map(&:join)
  Counter.get
end

Array.new(10, &test)
# => [81, 94, 98, 100, 100, 100, 98, 100, 97, 100]
```

Reply:
ugh, yeah, I see what's happening =P